src/main/java/core/basesyntax/service/impl/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.impl.service.CsvFileWriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvFileWriterServiceImpl implements CsvFileWriterService {
+    private static final String FILE_PATH = "src/main/resources/report.csv";
+    private static final String EXCEPTION_INFO = "Cannot write file";
+    private static final String FIRST_FILE_ROW = "fruit,quantity";
+    private static final String INVALID_INPUT_PARAMETER = "Invalid input parameter in writeData()";
+
+    @Override
+    public void writeData(String report) {
```

Comment:
Better to get the path as a parameter
```suggestion
    public void writeData(String report, String outputPath) {
```